src/common/marlin_server.cpp

```diff
@@ -502,12 +506,13 @@ static void _server_print_loop(void) {
         gcode.reset_stepper_timeout(); //prevent disable axis
         break;
     case mpsResuming_Begin:
-        if (marlin_server_print_reheat_ready()) {
-            marlin_server_unpark_head();
-            marlin_server.print_state = mpsResuming_UnparkHead;
+        if (marlin_server.vars.fan_check) {
+            if (fanctl1.getState() == CFanCtl::running) {
```

Comment:
/question/ Not sure I follow the logic here. Does this mean you can't *resume* if the fan isn't running?